river/feature_extraction/vectorize.py

```diff
@@ -59,7 +62,9 @@ def find_ngrams(tokens: typing.List[str], n: int) -> typing.Iterator[N_GRAM]:
     return zip(*[tokens[i:] for i in range(n)])
 
 
-def find_all_ngrams(tokens: typing.List[str], ngram_range: range) -> typing.Iterator[N_GRAM]:
+def find_all_ngrams(
+    tokens: typing.List[str], ngram_range: range
+) -> typing.Iterator[N_GRAM]:
```

Comment:
I think you have some code-style issues. We're now using `pre-commit` to enforce consistency. See the [contribution guide](https://github.com/online-ml/river/blob/master/CONTRIBUTING.md) to find out how to set this up locally :)